### ISSUE TITLE
query multiple characters by ID in graphql

### ISSUE THREAD

**Issue by Ey-Jay** [afuh/rick-and-morty-api#89]
Currently, there is no way to pass an array of ids to filter out characters like in the REST API, where you can pass multiple ids as params. Am I missing something?

**Comment by afuh** [comment]
That's correct. Since `character` is going to return one character, you can't pass an array of Ids.

**Comment by Ey-Jay** [comment]
I've done a PR that aims to provide this feature 🚀
